pkg/opssight/scanner.go

```diff
@@ -175,10 +179,12 @@ func (p *SpecConfig) imageFacadeVolumes() ([]*components.Volume, error) {
 	}
 	vols = append(vols, vol)
 
-	vols = append(vols, components.NewHostPathVolume(horizonapi.HostPathVolumeConfig{
-		VolumeName: "dir-docker-socket",
-		Path:       "/var/run/docker.sock",
-	}))
+	if !strings.EqualFold(p.config.ScannerPod.ImageFacade.ImagePullerType, "skopeo") {
```

Comment:
You may want to flip the logic for this test to `if p.config.ScannerPod.ImageFacade.ImagePullerType == "docker"` so it's a positive